### PR TITLE
Emoji: Fix twemoji incorrectly parsing characters in a few places.

### PR DESCRIPTION
**Pull request by pento**
- In the Editor: Image attributes were not begin correctly set, so the editor was saving emoji as images, instead of characters.
- In Emojify and the Editor: Overly aggressive character replacements were replacing basic characters that can be handled by all browsers.
- Reader: The full post view was using a custom emoji parser, instead of Emojify, so couldn't use the fixes from Emojify.

See #14887.

### CODE REVIEW

**Comment by blowery** [comment]
Who reviewed this?

File: client/blocks/reader-full-post/index.jsx [hunk 1]
```diff
@@ -77,21 +77,12 @@ export class FullPostView extends React.Component {
 
 	hasScrolledToCommentAnchor = false;
 
-	componentWillMount() {
-		const self = this;
-		asyncRequire( 'twemoji', twemoji => {
```
Comment (blowery) [inline client/blocks/reader-full-post/index.jsx]: iirc, twemoji isn't small. how much did this alter bundle sizes?

File: client/blocks/reader-full-post/index.jsx [hunk 2]
```diff
@@ -369,84 +346,92 @@ export class FullPostView extends React.Component {
...
-							<FeaturedImage src={ post.featured_image } /> }
-						{ isLoading && <ReaderFullPostContentPlaceholder /> }
-						{ post.use_excerpt
-							? <PostExcerpt content={ post.better_excerpt ? post.better_excerpt : post.excerpt } />
-							: <EmbedContainer>
-									<AutoDirection>
-										<div
-											className="reader-full-post__story-content"
-											dangerouslySetInnerHTML={ { __html: post.content } }
+					<Emojify>
```
Comment (blowery) [inline client/blocks/reader-full-post/index.jsx]: yay, much nicer than doing the parser work via lifecycle methods! Emojify adds a div, did that alter the reader display at all?

Comment (blowery) [inline client/blocks/reader-full-post/index.jsx]: appears to be fine in moderate testing

**Comment by pento** [comment]
Lack of review is my bad - while fixing this, I realised that the first point (Editor saving emoji as images instead of characters) was pretty terrible, and needed to be pushed out immediately.

Unfortunately, poor judgement on my part (after a couple of weeks of struggling with Calypso architecture and code reusability issues) meant I added the other changes without considering that they could easily wait for a proper review.

File: client/blocks/reader-full-post/index.jsx (hunk 1 above)
Comment (pento) [inline client/blocks/reader-full-post/index.jsx]: I don't know how to measure bundle size changes. 🙂

Twemoji is around 7KB gzipped by itself, I don't know if it's any better than that in the bundle. The bulk of Twemoji is in the images, which are only downloaded as needed.

**Pull request merged**